robottelo/ui/contentviews.py

```diff
@@ -707,8 +707,12 @@ def fetch_version_packages(self, name, version_name):
         """Return a list of all the packages inside specific content view
         version"""
         self.click(self.version_search(name, version_name))
-        self.click(tab_locators['contentviews.tab_version_packages'])
+        packages_tab = self.wait_until_element(
+            tab_locators['contentviews.tab_version_packages'], timeout=5)
```

Comment:
CVV tabs now are only show if there's some content in them, so extra check for tab presence is required. Tabs are kinda weird and are actually shown at first and then disappear (in about a second locally). I've set 5 secs as default timeout (12 secs) imo is a bit too much.

Reply:
do we have any approval for that strange behavior/bad javascript style?
Anyway, please re-run all tests that involve `fetch_version_packages` method calls in their body

Reply:
the change itself is intended, i've even tested it a bit and submitted 1 or 2 bzs for it. Regarding approval for a bad style - i don't think that's appropriate question to devs 🙂

Reply:
Turns out it's not that bad from automation perspective - page is not considered loaded while  tabs visibility isn't final. So after the click on cvv page we can continue with tabs as if nothing happened. I don't think we should rely on that fact though, so i'll add the checks, but it seems to be not that cool only from user perspective.